Appreciation.Function/Query/GetAppreciationsQuery.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using AppreciationApp.Core.Models;
+using AzureFromTheTrenches.Commanding.Abstractions;
+
+namespace AppreciationApp.Function.Query
+{
+    public class GetAppreciationsQuery : ICommand<List<Appreciation>>
+    {
+        public int DaysToLookBack { get; set; } = 7;
```

Comment:
needs to be 6 days look back, 7 days will include the previous Friday ( we want to go Saturday to Friday)

Reply:
This is just a default value, you can pass in whatever you want to the api with ?daysToLookBack=<some value>. That way from the frontend you could do last week, last month, last year even (assuming the 15Five API goes back that far).